apps/website/core-components/card/README.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Overview
+toc: true
+---
+
+Cards are used to apply a container around a related grouping of information.
```

Comment:
markdown content should be wrapped inside of the appropriate blocks to apply the correct core spacing tokens to the generated content. 
E.g:

``` markdown
::: component-summary
Summary of the component goes here.
:::
```

The same goes for the various level headings and associated description. Any one of the existing core-component md files should be following this pattern.